### DISCUSSION TITLE
How to change the time range of "Charge Details" graph?

### DISCUSSION THREAD

**Discussion by willdashwood** [teslamate-org/teslamate#3193] [Q&A] [answered]
Sorry if this has been covered before, I couldn't find any hits.

Is it possible to change the time range for the "Charge Details" graph? It appears to be the past 12 hours by default and I'd like it to be the past 3 hours.

Thanks!

**Comment by DrMichael** [comment, marked as answer]
You are not supposed to use that dashboard directly. Intention is to call it from the Charges dashboard, then it is filled with the time range of the charge.

**Reply by willdashwood** [reply]
Thank you, I should have known that really!
